packages/snap-preact-components/src/components/Molecules/Slideout/Slideout.test.tsx

```diff
@@ -126,6 +126,52 @@ describe('Slideout Component', () => {
 		expect(styles.background).toBe(args.overlayColor);
 	});
 
+	it('can set custom slideDirection', () => {
+		const args = {
+			active: true,
+			slideDirection: 'right' as SlideDirectionType,
+		};
+		const rendered = render(<Slideout {...args} />);
+		const overlayElement = rendered.container.querySelector('.ss__slideout.ss__slideout--active');
+		const styles = getComputedStyle(overlayElement);
+		expect(styles.right).toBe('0px');
+		expect(styles.left).toBe('');
+	});
+
+	it('can set custom transitionSpeed', () => {
+		const args = {
+			active: true,
+			transitionSpeed: '0.55s' as SlideDirectionType,
```

Comment:
I don't think this typing should be here, although it's probably not causing an issue.